yucca/image_processing/transforms/BiasField.py

```diff
@@ -3,16 +3,25 @@
 
 
 class BiasField(YuccaTransform):
-    def __init__(self, data_key="image", p_per_sample=1):
+    def __init__(
+        self,
+        data_key="image",
+        p_per_sample=1,
+        clip_to_input_range=False,
+    ):
         self.data_key = data_key
         self.p_per_sample = p_per_sample
+        self.clip_to_input_range = clip_to_input_range
 
     @staticmethod
     def get_params():
         # No parameters to retrieve
         pass
 
     def __biasField__(self, imageVolume):
+        mn = imageVolume.min()
```

Comment:
i would call this `min` and `max`. The cost in readability for saving one character is not worth it imo :-)